### ISSUE TITLE
More privacy for the profile image

### ISSUE THREAD

**Issue by BennySamir** [wireapp/wire#90]
Hello,

it would be nice if I could hide my profile picture from noncontacts. If I am looking for contacts, I can see the profile pictures of all people.

Signal has gone an interesting way. This is of course a bit more drastic, but in the sense of privacy it would also be very welcome at Wire: https://www.engadget.com/2017/09/07/signal-encrypted-profile-beta-chat-app/

Regards

**Comment by teller** [comment]
There are currently no plans to change the way we treat profile info, including the profile pic.

**Comment by IpsmLorem** [comment]
The signal one is the way to go

**Comment by rschobbert** [comment]
> There are currently no plans to change the way we treat profile info, including the profile pic

That's imho not a good idea. For me that is one of the main concerns, to not suggest the app to other people at the moment. Even Whatsapp is more secure in that way, because people only find you if they have your telephone number. If everybody sees the image while searching a user, especially good looking women and children should not use a picture of themselves.

Please rethink this issue, since it's concerning privacy.

Threema in my opinion has implemented it in a perfect way. It has three options for the profile pic: show to nobody (the default), to everybody you're writing to (shows the profile pic after you have written the first message), only to certain contacts (let's you choose which contacts can see your profile pic).

**Comment by baimafeima** [comment]
I second that. Deciding what to reveal to whom and when is mandatory in my opinion, especially when you call Wire a privacy-oriented service. I highly recommend to reconsider this.

**Comment by cardassian-tailor** [comment]
I agree. Give the user the option to determine what is viewable by non contacts. 

I think a great alternative that may fix the issue for both sides is this. Non contacts can only see the default wire profile images. Like the ones of mountians, flowers, etc. If you have a custom uploaded profile image - that is viewable only to contacts you are connected with. 

OR

If you upload/set a custom profile image - you have to choose between having it viewable by everyone, or just contacts.
